From 1a0e07f0a87c64a09538a5ec68bc66aef0f9277d Mon Sep 17 00:00:00 2001
From: Pooja Prajod <a0132412@ti.com>
Date: Thu, 21 May 2015 21:18:35 +0530
Subject: [gst-ducati] gst-ducati: Enable ducati to accept streams without
 profile and framerate info in caps

The framerate and profile info in caps is not necessary
for ducati plugin. Moreover eliminating these caps helps in
running elementary streams where h264parse fails to add these
info to caps.

Signed-off-by: Pooja Prajod <a0132412@ti.com>
---
 src/gstducatih264dec.c |    5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/src/gstducatih264dec.c b/src/gstducatih264dec.c
index d196fc3..962ceb4 100644
--- a/src/gstducatih264dec.c
+++ b/src/gstducatih264dec.c
@@ -85,15 +85,12 @@ static GstStaticPadTemplate sink_factory = GST_STATIC_PAD_TEMPLATE ("sink",
         "stream-format = byte-stream, "   /* only byte-stream */
         "alignment = au, "          /* only entire frames */
         "width = (int)[ 16, 2048 ], "
-        "height = (int)[ 16, 2048 ], "
-        "framerate = (fraction)[ 0, max ],"
-        "profile = (string){constrained-baseline, baseline, main, extended};"
+        "height = (int)[ 16, 2048 ]; "
         "video/x-h264, "
         "stream-format = byte-stream, "   /* only byte-stream */
         "alignment = au, "          /* only entire frames */
         "width = (int)[ 16, 2048 ], "
         "height = (int)[ 16, 2048 ], "
-        "framerate = (fraction)[ 0, max ],"
         "profile = (string) {high, high-10-intra, high-10, high-4:2:2-intra, "
         "high-4:2:2, high-4:4:4-intra, high-4:4:4, cavlc-4:4:4-intra}, "
         "level = (string) {1, 1b, 1.1, 1.2, 1.3, 2, 2.1, 2.2, 3, 3.1, 3.2, 4, 4.1, 4.2, 5, 5.1};")
-- 
1.7.9.5

